src/System.Net.Http/tests/FunctionalTests/HttpCookieProtocolTests.cs

```diff
@@ -36,6 +36,14 @@ private static CookieContainer CreateSingleCookieContainer(Uri uri, string cooki
 
         private static string GetCookieHeaderValue(string cookieName, string cookieValue) => $"{cookieName}={cookieValue}";
 
+        [Fact]
+        public void HttpClientHandler_SetNullCookieContainer_ThrowsArgumentNullException()
```

Comment:
This test should be in the HttpClientHandlerTest.cs class/file.

And the test name should be something like "CookieContainer_SetNull_ThrowsArgumentNullException"  This follows the principle that the first part of the test method name should be the method or property that is being tested.